### PR TITLE
feat: add re-recover and jq to pd

### PR DESCRIPTION
**Pull request by lijie0123**
Why:
- according to the add component plan

### CODE REVIEW

**Review by purelind — APPROVED** [review]
/lgtm

File: dockerfiles/pd.Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
-FROM hub.pingcap.net/bases/pd-base:v1.6.0
+FROM hub.pingcap.net/bases/pd-base:v1.6.1
 COPY pd-server /pd-server
 COPY pd-ctl /pd-ctl
+COPY pd-recover /pd-recover
```
Comment (wuhuizuo) [inline dockerfiles/pd.Dockerfile:4]: Please update dockerfiles in multi stage folder, thanks.

Comment (lijie0123) [reply dockerfiles/pd.Dockerfile:4]: done

**Comment by wuhuizuo** [comment]
- [ ] update packges.yaml.tmpl file

**Pull request merged**